cab-token-generator/java/com/google/auth/credentialaccessboundary/ClientSideCredentialAccessBoundaryFactory.java

```diff
@@ -206,4 +491,27 @@ public ClientSideCredentialAccessBoundaryFactory build() {
       return new ClientSideCredentialAccessBoundaryFactory(this);
     }
   }
+
+  @VisibleForTesting
+  String getAccessBoundarySessionKey() {
+
+    return intermediateCredentials != null
+        ? intermediateCredentials.accessBoundarySessionKey
+        : null;
+  }
+
+  @VisibleForTesting
+  AccessToken getIntermediateAccessToken() {
```

Comment:
nit: Just to stay consistent with the style of our other classes, can the order be:

Main Class stuff
Utility Static Classes
Builder Stuff

So move these getters above the utility/builder classes and move the NewBuilder method down with the rest of the builder logic